### PR TITLE
CI: CUDA 11.2 on dom

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

File: tests/regression/c_bindings/implementation.h [hunk 1]
```diff
@@ -1,12 +1,3 @@
-/*
- * GridTools
- *
- * Copyright (c) 2014-2021, ETH Zurich
- * All rights reserved.
- *
- * Please, refer to the LICENSE file in the root directory.
- * SPDX-License-Identifier: BSD-3-Clause
- */
```
Comment (fthaler) [inline tests/regression/c_bindings/implementation.h:9]: Just wondering about these changes, seem to be totally unrelated, but don’t really care.

**Review by fthaler — APPROVED** [review]

File: tests/regression/c_bindings/implementation.h (hunk 1 above)
Comment (havogt) [reply tests/regression/c_bindings/implementation.h:9]: Yes, these generated files got accidentally the license header. Just sneaked it in here.

**Pull request merged**